### ISSUE TITLE
[Bug]: ApplicationService.getAppContext  java.lang.NullPointerException

### ISSUE THREAD

**Issue by techday84** [OneSignal/react-native-onesignal#1660]
### What happened?

When I tested the app It seemed that everything is ok. In my test I received the push notifications in foreground, background and with the app closed, but when I published the app I started to receive error messages in crashlytics.

### Steps to reproduce?

```Markdown
"react-native-onesignal": "^5.0.6",
  "react": "18.2.0",
  "react-native": "0.73.4"

My android/build.gradle file:
...
buildscript {
    ext {
        buildToolsVersion = "33.0.0"
        minSdkVersion = 24
        compileSdkVersion = 33
        targetSdkVersion = 33
...
```

### What did you expect to happen?

When I tested the app it seemed that everything is ok but I am receiving crashlytics error messages now.

### React Native OneSignal SDK version

"react-native-onesignal": "^5.0.6",

### Which platform(s) are affected?

- [ ] iOS
- [X] Android

### Relevant log output

```Shell
ApplicationService.getAppContext
java.lang.NullPointerException

Fatal Exception: java.lang.RuntimeException
Unable to start receiver com.onesignal.notifications.receivers.FCMBroadcastReceiver: java.lang.reflect.InvocationTargetException
android.app.ActivityThread.handleReceiver (ActivityThread.java:4903)

com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1067)
Caused by java.lang.reflect.InvocationTargetException
java.lang.reflect.Constructor.newInstance0 (Constructor.java)

com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1067)
Caused by java.lang.NullPointerException
com.onesignal.core.internal.application.impl.ApplicationService.getAppContext (ApplicationService.kt:40)
com.onesignal.core.internal.preferences.impl.PreferencesService.getSharedPrefsByName (PreferencesService.kt:234)
com.onesignal.core.internal.preferences.impl.PreferencesService.get (PreferencesService.kt:134)
com.onesignal.core.internal.preferences.impl.PreferencesService.getString (PreferencesService.kt:42)
com.onesignal.common.modeling.ModelStore.load (ModelStore.kt:159)
com.onesignal.common.modeling.SimpleModelStore.<init> (SimpleModelStore.kt:23)
com.onesignal.core.internal.config.ConfigModelStore.<init> (ConfigModelStore.kt:8)
java.lang.reflect.Constructor.newInstance0 (Constructor.java)

java.lang.reflect.Constructor.newInstance (Constructor.java:343)
com.onesignal.common.services.ServiceRegistrationReflection.resolve (ServiceRegistration.kt:89)
com.onesignal.common.services.ServiceProvider.getServiceOrNull (ServiceProvider.kt:79)
com.onesignal.common.services.ServiceProvider.getService (ServiceProvider.kt:67)
com.onesignal.internal.OneSignalImp.initWithContext (OneSignalImp.kt:517)
com.onesignal.OneSignal.initWithContext (OneSignal.kt:208)
com.onesignal.notifications.receivers.FCMBroadcastReceiver.onReceive (FCMBroadcastReceiver.kt:26)
android.app.ActivityThread.handleReceiver (ActivityThread.java:4894)

com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1067)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by techday84** [comment]
In my App.js file:
import settings from './settings/settings';
export default function App() {
  useEffect(() => {
    OneSignal.initialize(settings.onesignal_configuration_code);
    OneSignal.Notifications.requestPermission(false);

  }, []);

**Comment by jkasten2** [comment]
@techday84 Thanks for reporting, this is a duplicate of issue #1587. The issue should be fixed in [Release 5.1.0](https://github.com/OneSignal/react-native-onesignal/releases/tag/5.1.0).

If you still see this issue after updating please follow up in issue #1587, thanks!

**Comment by uasghar-smile** [comment]
Hi @jkasten2 please check https://github.com/OneSignal/react-native-onesignal/issues/1587#issuecomment-2009117498
